examples/wesleycsj-helloWorld.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ *	Author: WesleyCSJ
+ *  Description: Render four colors on screen with 2s delay
+ *
+ *
+ */
+#include <iostream>
+#include <SDL2/SDL.h>
+#include "GameLoop.hpp"
+#include "InputHandler.hpp"
+
+#define SCREEN_WIDTH  800
+#define SCREEN_HEIGHT 600
+
+std::map<const char*, SDL_Keycode> input_config = {
+    {"left", SDLK_LEFT},
+    {"right", SDLK_RIGHT},
```

Comment:
The only necessary key used in this example is quit. Rest can be deleted.

Reply:
Ok, i didn't see 'quit' at middle of the matrix.